main.yaml

```diff
@@ -1,48 +1,41 @@
 features:
-  - title: What is QMC-HAMM?
+  - title: Dense hydrogen diffusion Monte Carlo (DMC) database
     name: what-wt
-    content: |
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
-      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
-      est laborum.
+    content:
+      Many aspects of the phase diagram of dense hydrogen remain poorly understood, 
+      sometimes even qualitatively. Dense hydrogen is predicted become a high-temperature 
+      superconductor at sufficiently high pressure and is crucial in determining the 
+      structure of gas giant planets. Addressing the entire phase diagram with accurate 
+      ab initio simulations like diffusion Monte Carlo (DMC) is not currently feasible 
```

Comment:
is diffusion usually lowercased, when prefacing an acronym?

Reply:
I think the lowercase is okay here (just following the conventions we have been using in our other written texts). For instance, we use this in our recent work "Stable solid molecular hydrogen above 900K from a machine-learned potential trained with diffusion Quantum Monte Carlo".